docs/site/content/en/docs/Reference/APIFilterQueryReference.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Horreum API Filter Query Reference
+description: Reference guide for API Filter Query
+date: 2024-03-27
+weight: 3
+---
+
+In Horreum the API provides the ability to filter results. Used when HTTP API response content type is `application-json`. The `API Filter Query` operates on the server side to reduce payload size.
+
+API operations that support Filter Query provide an HTTP request parameter named *query*. The value of the *query* parameter is a `JSONPath` expression. Any syntax errors in the `JSONPath` will cause the operation to fail with details of the  syntax issue.
+
+To use the Filter Query with an operation that returns a [Run](/docs/concepts/core-concepts/#run) data can be done as follows.
+
+```bash
+curl -s  'http://localhost:8080/api/sql/1/queryrun?query=$.*&array=true'  -H 'content-type: application/json' -H 'Authorization: Bearer '$TOKEN
+{"valid":true,"jsonpath":"$.*","errorCode":0,"sqlState":null,"reason":null,"sql":null,"value":"[\"urn:acme:benchmark:0.1\", [{\"test\": \"Foo\", \"duration\": 10, \"requests\": 123}, {\"test\": \"Bar\", \"duration\": 20, \"requests\": 456}], \"defec8eddeadbeafcafebabeb16b00b5\", \"This gets lost by the transformer\"]"}
```

Comment:
Can the output JSON be formatted so it is split over multiple lines? atm it is a long single line in the rendered doc

Reply:
sure

Reply:
I expanded the outer block of JSON. Do you want the Inner block also expanded ?

Reply:
I think it is ok to leave it as a string, as the output is not a valid JSON node, but a string represenation